tests/test_invoices.py

```diff
@@ -54,9 +54,9 @@ def test_invoice(node_factory, chainparams):
     assert 'routes' not in b11
     assert 'warning_capacity' in inv
 
-    # Make sure no wumbo invoices
-    with pytest.raises(RpcError, match=r'msatoshi cannot exceed 4294967295msat'):
-        l2.rpc.invoice(4294967295 + 1, 'inv3', '?')
+    # Make sure wumbo invoices warn about mpp being needed.
+    inv = l2.rpc.invoice(4294967295 + 1, 'inv3', '?')
+    assert 'warning_mpp' in inv
     l2.rpc.invoice(4294967295, 'inv3', '?')
```

Comment:
From Github actions

```
/home/runner/work/lightning/lightning/contrib/pyln-client/pyln/client/lightning.py:388>]
test_invoice failed; it passed 0 out of the required 1 times.
	<class 'pyln.client.lightning.RpcError'>
	RPC call failed: method: invoice, payload: {'msatoshi': 4294967295, 'label': 'inv3', 'description': '?'}, error: {'code': 900, 'message': "Duplicate label 'inv3'"}
	[<TracebackEntry /home/runner/work/lightning/lightning/tests/test_invoices.py:60>, <TracebackEntry /home/runner/work/lightning/lightning/contrib/pyln-client/pyln/client/lightning.py:845>, <TracebackEntry /home/runner/work/lightning/lightning/contrib/pyln-testing/pyln/testing/utils.py:623>, <TracebackEntry /home/runner/work/lightning/lightning/contrib/pyln-client/pyln/client/lightning.py:388>]
```

This is a bit strange, why we don't see it in other tests?

Reply:
Because we weren't specific enough about *how* it should error!